data/mods/Aftershock/items/armor/modular_exos/modular_exoskeleton_plating.json

```
@@ -0,0 +1,450 @@
+    "looks_like": "exo_arm_plastic",
+    "material_thickness": 3,
+    "armor": [ { "encumbrance": 16, "coverage": 0, "covers": [ "torso" ] } ]
+  },
+  {
+    "id": "exo_arm_vcc",
+    "type": "ARMOR",
+    "category": "armor",
+    "copy-from": "exo_arm_steel",
+    "name": { "str": "vacuum-cast carbide exoskeleton arm plate" },
+    "description": "Designed to attach to an exoskeleton's arms, these are sturdy plates of vacuum-cast carbide.",
+    "weight": "33000 g",
+    "volume": "5500 ml",
+    "material": [ "vacuum_carbide" ],
+    "material_thickness": 6,
+    "non_functional": "destroyed_exo_arm_vcc_plate"
+  },
+  {
+    "id": "destroyed_exo_arm_vcc_plate",
+    "type": "ARMOR",
+    "copy-from": "exo_arm_vcc",
+    "name": { "str": "shattered vacuum-cast carbide arm plate" },
+    "description": "The remains of a vacuum-cast carbide arm plate after taking a strong impact.  No longer provides functional protection, just weight.",
+    "looks_like": "exo_arm_vcc",
+    "armor": [ { "encumbrance": 16, "coverage": 0, "covers": [ "torso" ] } ]
+  },
+  {
+    "id": "exo_glove_steel",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "steel exoskeleton glove plate" },
+    "description": "Designed to attach to an exoskeleton's glove, this is a sturdy plate of steel.",
+    "weight": "4293 g",
+    "volume": "1 ml",
+    "material": [ "steel" ],
+    "symbol": "[",
+    "looks_like": "gloves_plate",
+    "color": "light_gray",
+    "warmth": 20,
+    "sided": true,
+    "material_thickness": 6,
+    "flags": [ "CANT_WEAR", "ABLATIVE_LARGE", "EXO_GLOVE_PLATE" ],
+    "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "hand_l", "hand_r" ] } ]
+  },
+  {
+    "id": "exo_glove_plastic",
+    "type": "ARMOR",
+    "category": "armor",
+    "copy-from": "exo_glove_steel",
+    "name": { "str": "plastic exoskeleton glove plate" },
+    "description": "Designed to attach to an exoskeleton's glove, this is a plate of plastic.",
+    "weight": "429 g",
+    "volume": "1 ml",
+    "material": [ "plastic" ],
+    "material_thickness": 3,
+    "non_functional": "destroyed_exo_glove_plastic_plate"
+  },
+  {
+    "id": "destroyed_exo_glove_plastic_plate",
+    "type": "ARMOR",
+    "copy-from": "exo_glove_plastic",
+    "name": { "str": "shattered plastic leg plate" },
+    "description": "The remains of a plastic leg plate after taking a strong impact.  No longer provides functional protection, just weight.",
+    "looks_like": "exo_glove_plastic",
+    "armor": [ { "encumbrance": 20, "coverage": 0, "covers": [ "hand_l", "hand_r" ] } ]
+  },
+  {
+    "id": "exo_glove_vcc",
+    "type": "ARMOR",
+    "category": "armor",
+    "copy-from": "exo_arm_steel",
+    "name": { "str": "exoskeleton vacuum-cast composite glove plate" },
+    "description": "Designed to attach to an exoskeleton's glove, this is a sturdy plate of vacuum-cast composite.",
```

```suggestion
    "description": "Designed to attach to an exoskeleton's glove, this is a sturdy plate of vacuum-cast carbide.",
```
Fix this error in the other descriptions